docs/book/mlops-stacks/model-deployers/mlflow.md

```diff
@@ -69,7 +69,7 @@ class MLFlowDeploymentLoaderStepConfig(BaseStepConfig):
     step_name: str
     running: bool = True
     
-model_deployer = mlflow_deployer_step(name="model_deployer")
+model_deployer = mlflow_deployer_step(step="model_deployer")
```

Comment:
I think this is a different function argument that was renamed?

Reply:
True, good catch. Fixed it.